src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -797,7 +807,7 @@ private void ImportProcedure(string library, ProcedureNode proc)
                     }
 
                     return new TypedParameter(arg.Name, argType, defaultArgumentNode);
-                });
+                }).ToList();
```

Comment:
Disable lazy evaluation here. Otherwise getting default argument operation will be evaluated in the constructor of `FunctionDescriptor`.